broker/configuration/impl.go

```diff
@@ -0,0 +1,39 @@
+package configuration
+
+import (
+	"encoding/json"
```

Comment:
JSON? Didnt we agreed YAML for configs? (or did I just agreed with myself?)

Reply:
I think you agreed with yourself... This configuration file is going to contain maps of groups, accounts and roles. I really think JSON is more appropriate.

Reply:
No, we did agree on YAML, it wasn't Camilo's imagination :)